src/components/FormInput/FormInput.test.tsx

```diff
@@ -0,0 +1,88 @@
+import userEvent from '@testing-library/user-event';
+import { useState } from 'react';
+import { describe, it, expect, vi } from 'vitest';
+
+import { render, screen } from '../../test/test-utils';
+
+import { FormInput } from './FormInput';
+
+describe('FormInput', () => {
+  const defaultProps = {
+    label: 'Mail Id',
+    value: '',
+    onChange: vi.fn(),
```

Comment:
defaultProps defines 'onChange: vi.fn()' at module scope. This common mock will accumulate calls over the test suite unless reset. See above suggestion to create or reset mocks per test.

**Sugerencia:**
```typescript
Move vi.fn() creation into individual tests or a beforeEach and reference that mock inside each test rather than sharing a single mock instance globally.
```